hbase-server/src/test/java/org/apache/hadoop/hbase/quotas/TestDefaultOperationQuota.java

```diff
@@ -125,4 +141,134 @@ public void testScanEstimateShrinkingWorkload() {
     // shrinking workload should only shrink estimate to maxBBS
     assertEquals(maxBlockBytesScanned, estimate);
   }
+
+  @Test
+  public void testLargeBatchSaturatesReadNumLimit()
```

Comment:
Should we also have a `testLargeBatchSaturatesWriteNumLimit` ?

Reply:
Yeah good idea